tox.ini

```diff
@@ -21,7 +21,7 @@ commands =
     pip install git+https://github.com/pytoolz/cytoolz.git#egg=cytoolz==0.10.1.dev0
     pip install -e .[all]
     pip install -i https://test.pypi.org/simple/ fetch-p2p-api==0.0.1
-    pytest --doctest-modules aea packages tests/ --cov-report=html --cov-report=term --cov=aea --cov=packages/protocols --cov=packages/connections {posargs}
+    pytest --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
```

Comment:
@marcofavorito since you also imported the skills code into coverage checks we needed the `scikit-image` dependency. But we don't want to run coverage on skill code atm as we have no tests other than the end-to-end tests.

Reply:
I see. thanks